doc/dev/installation.rst

```diff
@@ -414,3 +414,123 @@ To generate html files for the documentation for Catalyst:
+  git clone --recurse-submodules --shallow-submodules https://github.com/PennyLaneAI/catalyst.git
+
+  # Activate the development environment based on the latest TestPyPI wheels (please provide a name)
+  cd catalyst
+  . ./activate_dev_from_wheel.sh myenv
+
+To exit the virtual environment, type:
+
+.. code-block:: console
+
+  deactivate
+
+How Does it Work?
+^^^^^^^^^^^^^^^^^
+
+The ``activate_dev_from_wheel.sh`` script acts as a wrapper of the ``setup_dev_from_wheel.sh`` script,
+which performs the following steps:
+
+- Create (if not created yet) and activate a Python virtual environment in the ``/tmp`` folder
+  with the provided name. This way the system Python configurations do not get affected, nor
+  other virtual environments:
+
+  .. code-block:: console
+
+    python3 -m venv /tmp/myenv
+    source /tmp/myenv/bin/activate
+
+- As a clean-up measure, Uninstall any previous Catalyst instances of that particular virtual
+  environment:
+
+  .. code-block:: console
+
+    python -m pip uninstall -y pennylane-catalyst
+
+- Install the latest CPL suite wheels from the TestPyPI server:
+
+  .. code-block:: console
+
+    python -m pip install --extra-index-url https://test.pypi.org/simple/ pennylane pennylane-lightning pennylane-catalyst --pre --upgrade
+
+- Search in the Git repository for the latest commit whose version corresponds to the 
+  Catalyst wheel version. This will leave the repository at a "detached head" state.
+
+- Once the Catalyst wheel and the repository are at the same version, create hard links from 
+  the Catalyst wheel code to the frontend code of the repository, in order to allow working
+  directly with the frontend repository codebase and at the same time test the changes while
+  using the installed Catalyst wheel libraries, hence avoiding compilation:
+
+  .. code-block:: console
+
+    export SITEPKGS=$(python3 -c 'import sysconfig; print(sysconfig.get_paths()["purelib"])')
+    export CATALYST_WHEEL=$SITEPKGS/catalyst
+    cp -lrf $CATALYST_WHEEL .frontend/
+
+Further Steps
+^^^^^^^^^^^^^
+
+If everything goes well, ``git status`` should not report any changed files. 
+
+Before making changes to the frontend, make sure you create a new branch:
+
+.. code-block:: console
+
+  git checkout -b new-branch-name
+
+Once in the new branch, make the wanted changes. Use the IDE of your preference.
+
+You can test the changes by executing your sample code under the same virtual environment you used
+with the scripts. As you are actually directly changing the code stored at the Python ``site-packages``
+folder, you will be automatically using the shared libraries provided by the Python wheels. Again,
+there is no need to compile Catalyst from source.
+
+When you are ready, commit your changes as usual. As your branch was created from a detached head,
+you can choose to rebase your branch according to the ``main`` branch:
+
+.. code-block:: console
+  
+  git fetch
+  git rebase -i origin/main
```

Comment:
I think this might be dangerous to suggest, because the compiled code might have changed between the wheel commit and main. I would suggest removing this.

Reply:
Removed. Done.